articles/webcam-matlab/index.md

```diff
@@ -0,0 +1,148 @@
+### WEBCAM AND IPCAM IN MATLAB
+Introduction
+A WEBCAM is connected to pc and can be used to make videos available to other users. The connection is normally through the USB. IPCAM involves the combination of high-end web functionality, pc and network interface to one network-ready product
+In this tutorial, we will look at how to install the WEBCAM and IPCAM hardware package for MATLAB which is necessary to access the hardware. We will also look at how to access the laptop's inbuilt WEBCAM and external USB webcam with Matlab code to take snapshots, preview, and record a video clip.
+The android application to convert mobile phone's camera as an IPCAM and look at the MATLAB code that can be used to make the mobile phone's camera as an IPCAM
+
+### Prerequisites
+- Matlab installed on your pc
+- Webcam and Ipcam support packages installed.
+- Webcam android application installed on your mobile phone.
+
+### Need for camera interface.
+There are several projects which require either image or videos as primary input when they are implemented. For example;
+1. Face identification and recognition.
+2. Face recognition-based security system and attendance system.
+3. Facial expression recognition.
+4. Object identification and classification.
+5. Fire detection on site.
+6. Vehicle license plate recognition and vehicle logging system.
+7. and others...
+
+### How to install the hardware support package for Matlab.
+-Log in to your math work account at the top right corner of the Matlab window.
+![sign in tab](/engineering-education/webcam-matlab/signin.jpg)
+
+-Click on the `adds on` drop-down arrow and select the `get hardware support package`.
+![add_on menu](/engineering-education/webcam-matlab/adds_on.png)
+-The hardware support package opens up and on the search tab, type `Matlab support package for USB webcam`  and search.
+![support package](/engineering-education/webcam-matlab/webcam.png)
+-After locating it, select the support package and click on the install button.
+![install tab](/engineering-education/webcam-matlab/windows3_1.png)
+for our case the push button for install changes to manage since the package is already installed in our PC.
+Since we are accessing the IP camera, we need the IP support package.
+-Using the same process above, we will locate the `Matlab support package for IP camera` and then install it.
+![support package](/engineering-education/webcam-matlab/package.jpg)
+
+### Matlab code for accessing laptop’s inbuilt webcam and an external USB webcam.
+To see the list of all attached cameras to your laptop at the moment of use, execute the code below;
+```Matlab
+>>webcamlist
```

Comment:
Remove these `>>`.

It would help the readers to copy-paste the codes, if needed

Reply:
Do these changes for all the subsequent code snippets.